HelpSource/Classes/EnvironmentRedirect.schelp

```diff
@@ -53,8 +63,10 @@ EnvironmentRedirect implements some of the interface of link::Classes/Environmen
 
 method:: push, pop, clear, choose
 method:: make, use
-method:: do, keysValuesDo, sortedKeysValuesDo, putAll
-method:: keysValuesArrayDo, doFunctionPerform
+method:: do, keysValuesDo, sortedKeysValuesDo
```

Comment:
Same as above. I think these either need descriptions or links to where a description is given if it is inherited or implemented elsewhere.

Reply:
I think it could be marked as resolved.